sdk/servicebus/azure-servicebus/azure/servicebus/aio/management/_utils.py

```diff
@@ -0,0 +1,47 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+from typing import cast
+from xml.etree.ElementTree import ElementTree
+
+
+import urllib.parse as urlparse
+
+from azure.servicebus.management import _constants as constants
+from ...management._handle_response_error import _handle_response_error
+
+
+async def extract_data_template(feed_class, convert, feed_element):
```

Comment:
Are you sure this function is supposed to be async?

Reply:
Actually I just reviewed core and I think you're right

Reply:
Yes it works. It wouldn't work if I removed "async".
Let me know if there is a different or better way to do it.